pkg/apis/cyclone/v1alpha1/workflow_run.go

```diff
@@ -29,24 +29,37 @@ type WorkflowRunSpec struct {
 	StartStages []string `json:"startStages"`
 	// Stages in the workflow to end execution
 	EndStages []string `json:"endStages"`
-	// Maximum time this workflow should run
+	// Maximum time this workflow can run
 	Timeout string `json:"timeout"`
 	// ServiceAccount used in the workflow execution
 	ServiceAccount string `json:"serviceAccount"`
 	// Resource parameters
 	Resources []ParameterConfig `json:"resources"`
 	// Stage parameters
 	Stages []ParameterConfig `json:"stages"`
+	// Execution context which specifies namespace and PVC used
+	ExecutionContext *ExecutionContext `json:"executionContext"`
 }
 
 // ParameterConfig configures parameters of a resource or a stage.
 type ParameterConfig struct {
 	// Whose parameters to configure
 	Name string `json:"name"`
-	// Parameters
+	// Parameters ...
 	Parameters []ParameterItem `json:"parameters"`
 }
 
+// ExecutionContext defines execution context of a workflow, namespace, pvc
```

Comment:
could you please write `namespace, pvc cluster would be specified` in a new line to separate `... a workflow` with `namespace, pvc ...`